linux/Platform.c

```diff
@@ -1017,6 +1019,31 @@ bool Platform_init(void) {
    LibSensors_init();
 #endif
 
+   char target[PATH_MAX];
+   ssize_t ret = readlink(PROCDIR "/self/ns/pid", target, sizeof(target) - 1);
+   if (ret > 0) {
+      target[ret] = '\0';
+
+      if (!String_eq("pid:[4026531836]", target)) { // magic constant PROC_PID_INIT_INO from include/linux/proc_ns.h#L46
+         Running_containerized = true;
+         return true; // early return
+      }
+   }
+
+   FILE* fd = fopen(PROCDIR "/1/mounts", "r");
+   if (fd) {
+      char lineBuffer[256];
+      while (fgets(lineBuffer, sizeof(lineBuffer), fd)) {
+         // detect lxc or overlayfs and guess that this means we are running containerized
+         if (String_startsWith(lineBuffer, "lxcfs ") || String_startsWith(lineBuffer, "overlay ")) {
```

Comment:
Can you provide a copy of the `/proc/1/mounts` from outside the container on your system? If you happen to have docker installed: Can you also provide a copy what that looks like inside a docker container?

Reply:
Sure: https://gist.github.com/nbdd0121/8e5df3a29cf921b6b8c6007e87cfc653

I removed a few bind mounts otherwise it's a mess... The overlay fs that serves the root of the container is visible outside as well. I think checking for `"overlay / "` would be sufficient fix?

Reply:
@nbdd0121 This check seems to have been improved in 5558c015 already. Could you try a version that includes this patch (head of main branch will do)?

Reply:
Thanks for the pointer. Yes, the head of main branch works. Is there a date/plan for 3.2.3 release?